lib/geocoder/results/concerns/flexible_polyline_decoder.rb

```diff
@@ -0,0 +1,97 @@
+module Geocoder::Result
```

Comment:
where did this logic come from? was it from this repo?https://github.com/heremaps/flexible-polyline/

Reply:
Yes @sergioloza